### DISCUSSION TITLE
How to change language?

### DISCUSSION THREAD

**Discussion by InfoLibre** [rmottola/Arctic-Fox#221] [General]
I'm testing version 44.0b under Windows XP SP3. I don't find how to change the interface language. Could someone help me, please?

**Comment by InfoLibre** [comment]
Do you plan to release Artic Fox in other languages than english?

**Reply by rmottola** [reply]
I don't release language packs other than English and nobody has helped contributing one, but it would be possible if somebody wants to help!

**Comment by rmottola** [comment]
I also add that Windows builds are contributed up to now only by @roytam1, lately he had troubles and little time. A regular windows contributor for builds would help a lot.
